apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -175,7 +176,12 @@ export default {
           } catch (error) {
             this.errorPerColumn[column.id] = "Computation failed: " + error;
           }
-        } else if (this.applyDefaultValues && column.defaultValue) {
+        } else if (
+          this.applyDefaultValues &&
+          column.defaultValue &&
+          this.onlySetDefaultValueOnes
+        ) {
+          this.onlySetDefaultValueOnes = false;
```

Comment:
isn't it easier to do (this.internalValues[column.id] === null && this.applyDefaultValues && column.defaultValue)?

Reply:
Looking at the code a bit more I'd noticed that the default values are also set in the created() section. That doesn't seem correct to me.

Reply:
a, good eye, moving it to the create is a lot better idea.